### PR TITLE
fix(draggable): move target instead of handle

### LINKED ISSUE #84: Dragging only moves handle, leaves dialog behind
## I'm submitting a...

<pre><code>
[x] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

When trying to drag the drag handle ("marker") of a dialog, only the handle itself moves.

## Expected behavior

The whole dialog moves as it did before

## Minimal reproduction of the problem with instructions

1. clone this repo (the playground)
2. `npm ci && npm run start`
3. create a dialog with `draggable: true`
4. (optional) apply custom style to the drag handle to make it easier to see
5. try to drag the dialog

alternatively, the same behaviour is reproducible over on https://ngneat.github.io/dialog/
you can use your browser's css inspector to add something like this to make the issue more visible:
```css
.ngneat-drag-marker {
    background: red;
}
```
chrome_3CJw0oCXDy

## Environment

<pre><code>
Angular version: 15.0.0, 15.0.4 (almost latest)

Browser:
- [x] Chrome (desktop) version 108
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [x] Firefox version 108
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [x] Edge version 108
 

Others:
* I noticed the regression in a local project after upgrading from 2.1.1 to 3.3.0
* I was able to reproduce it in the latest playground (1d5509210d124d65c1cf15c7ee84951dfe1cde2c) aswell as 3.0.0 (028bccadd8275ef8966ff0921894c9fd32a12e9a)
* it's weird how the latest release commit on github is 3.1.0, but npm has 3.3.0 already. I didn't look into that any further and hope my version info is correct
</code></pre>

### PR DESCRIPTION
**Pull request by jemand771**
since the handle element is a child of the target element, it still gets moved as expected. this was probably more of a typo than an actual bug.

fixes #84

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] ~~Docs have been added / updated (for bug fixes / features)~~ n/a

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

tl;dr: dragging a dialog handle only moves the handle and leaves the dialog in place.
Issue Number: #84 

## What is the new behavior?
dragging the dialog handle moves the whole dialog (including the handle), as expected

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by jemand771** [comment]
I have no idea how to automatically test this. Since the draggable directive didn't have tests before and this bug was more of a typo than a logic issue, imo it's fine without tests (for now)

**Comment by NetanelBasal** [comment]
But what caused the issue? It worked in the past.

**Comment by jemand771** [comment]
see [this line](https://github.com/ngneat/dialog/commit/f5575f59da2fb61b9c0aa229e8890eaf32a2a56c#diff-fbaab3b279a963cabb7e804abe0e49a9008089b28e136a097b3955dd358e0991R141) in https://github.com/ngneat/dialog/commit/f5575f59da2fb61b9c0aa229e8890eaf32a2a56c (also linked in the issue)

in some [other places](https://github.com/ngneat/dialog/commit/f5575f59da2fb61b9c0aa229e8890eaf32a2a56c#diff-fbaab3b279a963cabb7e804abe0e49a9008089b28e136a097b3955dd358e0991L61) in that file, the (correct) change
```diff
- this.renderer.setStyle(this.handle, 'cursor', '');
+ this.handle.style.setProperty('cursor', '');
```
was made. but for the transform line, this happened:
```diff
-           this.renderer.setStyle(this.target, 'transform', transform);
+           this.handle.style.setProperty('transform', transform);
```
-> accidentally applying the transform to the handle instead of the target.

**Comment by NetanelBasal** [comment]
Thanks
